quinn-proto/src/connection/mod.rs

```diff
@@ -673,18 +675,25 @@ impl Connection {
         self.endpoint_events
             .push_back(EndpointEventInner::RetireResetToken(path_id));
 
+        let pto = self.pto_max_path(SpaceId::Data);
```

Comment:
Why `pto_max_path` and not `self.pto(SpaceId::Data, path_id)`?
(I don't suggest the latter, just don't know)